java/ql/test/library-tests/dataflow/collections/Test.java

```diff
@@ -84,7 +84,17 @@ public void run4() {
     p.put("key", tainted);
```

Comment:
This method is similar to `setProperty`, should I also change it like this?
```suggestion
    sink(p.put("key", tainted)); // Flow
```

Reply:
Done in https://github.com/github/codeql/pull/9963/commits/8eebe710ce568c49f57c51d958b211e4e0e4fa63